app/imports/api/users/methods.js

```diff
@@ -0,0 +1,35 @@
+import { Meteor } from 'meteor/meteor';
+import { ValidatedMethod } from 'meteor/mdg:validated-method';
+import {
+    setAdminSchema,
+    removeAdminSchema,
+} from './schema.js';
+
+
+export const setAdmin = new ValidatedMethod({
+    name: 'setAdmin',
+    validate: setAdminSchema.validator({ clean: true }),
+    run({ userId }) {
+        const user = Meteor.user().isAdmin;
```

Comment:
Change it to `const { isAdmin } = Meteor.user();`, because your `user` doesn't contain user...